lib/xandra/protocol.ex

```diff
@@ -26,6 +26,19 @@ defmodule Xandra.Protocol do
     end
   end
 
+  defmacrop decode_string_or_atom({:<-, _, [value, buffer]}, atom_keys?) do
+    quote do
+      {unquote(value), unquote(buffer)} =
+      if unquote(atom_keys?) do
```

Comment:
Bad indentation here. Indent the `if` one more level than the previous `=` line.

Reply:
✅